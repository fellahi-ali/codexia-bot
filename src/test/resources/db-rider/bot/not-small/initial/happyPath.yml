github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

codexia_project:
    -
        id: 6
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        project_created_at: "2019-12-20 12:01:02"
        badges: "newbie,L0"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "12"

github_repo_stat:
    -
        id: 4
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "LINES_OF_CODE"
        stat: >-
            {
              "type": "LINES_OF_CODE",
              "itemList": [
                {
                  "language": "TypeScript",
                  "files": 84,
                  "lines": 9415,
                  "blanks": 1200,
                  "comments": 682,
                  "linesOfCode": 3533
                },
                {
                  "language": "Total",
                  "files": 173,
                  "lines": 31482,
                  "blanks": 1446,
                  "comments": 706,
                  "linesOfCode": 29330
                }
              ]
            }
        created_at: "2020-02-11 10:06:16.123456"
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: >-
            {
                "type": "GITHUB_API",
                "language": "TypeScript"
            }
        created_at: "2020-02-16 18:46:09.652645"
    -
        id: 6
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "LINES_OF_CODE"
        stat: >-
            {
                "type": "LINES_OF_CODE",
                "itemList": [
                {
                    "language": "TypeScript",
                    "files": 84,
                    "lines": 9415,
                    "blanks": 1200,
                    "comments": 682,
                    "linesOfCode": 5000
                },
                {
                    "language": "Total",
                    "files": 173,
                    "lines": 31482,
                    "blanks": 1446,
                    "comments": 706,
                    "linesOfCode": 29330
                }
                ]
            }
        created_at: "2020-02-19 11:22:33.123456"

too_small_result:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        github_repo_stat_id: 4
        state: SET

codexia_review:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 6
        author: "TOO_SMALL"
        reason: "3533"
        text: The repo is too small (LoC is 3533).
