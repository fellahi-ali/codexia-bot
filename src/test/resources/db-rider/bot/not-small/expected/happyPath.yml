too_small_result:
    -
        github_repo_id: 1
        github_repo_stat_id: 4
        state: SET
    -
        github_repo_id: 1
        github_repo_stat_id: 6
        state: RESET

codexia_review:
    -
        codexia_project_id: 6
        author: "TOO_SMALL"
        reason: "3533"
        text: The repo is too small (LoC is 3533).
    -
        codexia_project_id: 6
        author: "TOO_SMALL"
        reason: "5000"
        text: The repo is not small anymore (LoC is 5000).
